src/dot/commons/utils.py

```diff
@@ -52,12 +52,41 @@ def find_files_from_path(path: str, ext: List):
         [
             files.extend(glob.glob(path + "**/*." + e, recursive=True)) for e in ext  # type: ignore
         ]
+        random.shuffle(files)
```

Comment:
please use `np.random` so that you can seed it. The random operations should be deterministic for reproducibility.